modules/150-user-authn/openapi/config-values.yaml

```
@@ -92,6 +92,67 @@ properties:
             A CA for accessing the API:
             * If the parameter is not set, Kubernetes CA is used.
             * We recommend using a self-signed certificate (and specify it as masterCA) if an HTTP proxy (that terminates HTTPS traffic) is used for exposing.
+  passwordPolicy:
+    type: object
+    default: null
+    required: ["id"]
+    properties:
+      id:
+        type: string
+        default: "default"
+        description: |
+          Unique policy ID.
+          
+          Reserved for future use.
```

```suggestion
      id:
        type: string
        default: "default"
        description: |
          Unique policy ID.
          
          Reserved for future use.
```

I think we do not need to reserve fields for the future use, they all reserved by default.